.github/workflows/check.yml

```diff
@@ -55,12 +59,15 @@ jobs:
       - uses: actions/checkout@v4
         with:
           submodules: true
-      - name: Install ${{ matrix.toolchain }}
-        uses: dtolnay/rust-toolchain@master
+
+      - name: Install rust ${{ matrix.toolchain }}
```

Comment:
```suggestion
      - name: Install ${{ matrix.toolchain }}
```

Reply:
I've changed name on purpose. It looks like that in ci logs currently:
<img width="221" alt="image" src="https://github.com/user-attachments/assets/5b8c30cb-efd7-4d83-9b2a-e2153e81a98b">

Reply:
Fair enough, thanks!